tests/constants.rs

```diff
@@ -0,0 +1,5 @@
+#[allow(dead_code)]
+pub const UNINITIALIZED_BYTE: u8 = 0xDE;
+
+#[allow(dead_code)]
```

Comment:
Ditto.

Reply:
I think the value of `UNINITIALIZED_USIZE` shouldn't have any zero bytes in it.

However, I suggest that we just remove the one use of `UNINITIALIZED_USIZE` we have for now and remove `UNINITIALIZED_USIZE`.